felix/fv/tiered_policy_test.go

```diff
@@ -342,8 +342,8 @@ var _ = infrastructure.DatastoreDescribe("connectivity tests with policy tiers _
 				}
 				return strings.Count(out0, "default.ep1-1-allow-all") > 0 &&
 					strings.Count(out1, "default.ep1-1-allow-all") == 0 &&
-					strings.Count(out0, "default/staged:default.np3-4") == 0 &&
-					strings.Count(out1, "default/staged:default.np3-4") > 0
+					strings.Count(out0, "staged") == 0 &&
```

Comment:
Could you explain this change?

Reply:
It's a more strict check. Instead of checking specific staged rule is programmed or not, this checks that no staged policy should be available in dataplane. Based on this PR, the expectation is that no staged policy should appear in dataplane rules.

Reply:
Thank you!